lib/tools/adb-emu-commands.js

```diff
@@ -18,29 +18,37 @@ emuMethods.isEmulatorConnected = async function (udid) {
   return false;
 };
 
+emuMethods.checkEmulatorConnected = async function (udid) {
```

Comment:
i think so. i think it's better for this function to error if the parameters you call it with would otherwise cause this function to do nothing. in this case it means erroring if you fail to pass in a udid.

if you wind up in a position where you don't have a udid, and want nothing to happen, then you just choose not to call this function. rather than call it and have it do nothing.